packages/terra-tag/src/Tag.jsx

```diff
@@ -33,6 +33,10 @@ const propTypes = {
    * Callback function triggered when key is released.
    */
   onKeyUp: PropTypes.func,
+  /**
+   * Callback function triggered when tag is clicked with mouse.
+   */
+  onTagClick: PropTypes.func,
```

Comment:
Is clicking the only way to activate this?  If so, ss there a specific reason we only care about the click event as it would not be accessible?

Reply:
Added additional description for this prop.
This is an Internal prop used to focus to current tag if the tag is clicked with mouse. And there are other props exposed to consumers to manage the events.
Thanks

Reply:
What happens if the current tag and focus and the user presses enter?  Also, if it is internal, why does it need to be a prop?